### PR TITLE
Refactor: use v3 form for the onboarding wizard

### PR DESCRIPTION
**Pull request by JoshuaHungDinh**
Resolves [GIVE-997]

## Description
These changes address replacing the v2 wizard form from the onboarding wizard with a v3 form. To facilitate this several changes were made. 

### Creating the form
The form creation processes has been updated in the `FormRepository` class so that a v3 form is created. On creation the v3 form includes a header image, an alt for the image & the image position ('above').

### Adding/Removing selected features for the form
To update the form fields, the logic has changed from manipulating form meta to the updating the various blocks needed in the v3 form. While removing the blocks from a form there can be an error when removing a block that is not available on the form. To accommodate this, a conditional was included inside the BlockCollection `remove` method.

### Displaying the form
The form url used to src the iframe for display has been updated. Inconsistencies with providing the form url  immediately through the window prior to form creation lead to issues while displaying. Due to this the base form preview url is still passed through the window but the formId used is now managed on the frontend and received after form creation from the route.

## Affects
Onboarding Wizzard

## Testing Instructions
- Go through the onboarding wizard which can be found either by going to Donations-setup from the left sidebar then selecting "First Time Configuration" from the page or through a brand new give site.
- Try adding features from the wizard and verify they show up on the wizard form.
- Try navigating back and removing features and verify the removed features do NOT display.

## Pre-review Checklist
-   [x] Acceptance criteria satisfied and marked in related issue
-   [x] Relevant `@unreleased` tags included in DocBlocks
-   [ ] Includes unit tests
-   [ ] Reviewed by the designer (if follows a design)
-   [x] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

[GIVE-997]: https://stellarwp.atlassian.net/browse/GIVE-997?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: assets/src/images/admin/onboarding/header-image.png
Comment (pauloiankoski) [inline assets/src/images/admin/onboarding/header-image.png:1]: Could we use a JPEG instead so that we have a smaller file size?

**Review by pauloiankoski — CHANGES_REQUESTED** [review]

File: assets/src/images/admin/onboarding/header-image.png
Comment (JoshuaHungDinh) [reply assets/src/images/admin/onboarding/header-image.png:1]: Resolved: 6f504a0c15cc867e9bc7637056f793ca4fb3713d

File: src/Onboarding/FormRepository.php [hunk 1]
```diff
@@ -66,17 +72,30 @@ protected function isFormAvailable($formID)
...
      */
     protected function makeAndPersist()
     {
-        $formID = $this->defaultFormFactory->make();
+        $form = new DonationForm([
+            'title'    => __('GiveWP Donation Form', 'give'),
+            'status'   => DonationFormStatus::PUBLISHED(),
+            'settings' => FormSettings::fromArray([
+                'designId' => 'multi-step',
+                'designSettingsImageUrl' => GIVE_PLUGIN_URL . '/assets/dist/images/admin/onboarding/header-image.png',
```
Comment (pauloiankoski) [inline src/Onboarding/FormRepository.php:87]: Remember to update it here too.

**Review by pauloiankoski — APPROVED** [review]
Left one last comment but then it's ready to merge

File: src/Onboarding/FormRepository.php (hunk 1 above)
Comment (JoshuaHungDinh) [reply src/Onboarding/FormRepository.php]: Good catch, thanks! 2cf25936ee4ef5c9e6d3b6c41a4d009bce86de8c

**Pull request merged**